css/styles.css

```diff
@@ -10229,13 +10230,13 @@ header.masthead  {
 
 .banner {
   padding: 1em;
-  margin: 0;
+  margin: 0 0 3em 0;
   background-color: var(--pf-white) !important;
   border-style: solid;
-  border-radius: 1px;
-  border-color: var(--pf-deep-aqua);
+  border-width: 1px;
```

Comment:
@GTLook this is why the border looked wider than 1px - it was! `border-width` vs. `border-radius` 🤯